board/opendingux/package/libjpeg8/libjpeg8.mk

```diff
@@ -0,0 +1,21 @@
+################################################################################
+#
+# libjpeg
+#
+################################################################################
+
+LIBJPEG8_VERSION = 8d
+LIBJPEG8_SITE = http://www.ijg.org/files
+LIBJPEG8_SOURCE = jpegsrc.v$(LIBJPEG8_VERSION).tar.gz
+LIBJPEG8_LICENSE = IJG
+LIBJPEG8_LICENSE_FILES = README
+LIBJPEG8_INSTALL_STAGING = YES
+
+define LIBJPEG8_REMOVE_USELESS_TOOLS
+	rm -f $(addprefix $(TARGET_DIR)/usr/bin/,cjpeg djpeg jpegtran rdjpgcom wrjpgcom)
+endef
```

Comment:
Don't do that here, instead remove them in the post-build script.

Reply:
This is copied verbatim from the buildroot libjpeg package. Are you sure you want to diverge?

Reply:
Besides, doing it here is better. We only need the library, never the binaries, so there's not a good reason to put it into the post-build script.